server/tests/integration_test.rs

```diff
@@ -455,7 +455,7 @@ fn start_modelardbd(path: &Path) -> Child {
 
     // The thread needs to sleep to ensure that the server has properly started before sending
     // streams to it.
-    thread::sleep(Duration::from_secs(2));
+    thread::sleep(Duration::from_secs(5));
```

Comment:
The randomly failing integration tests may be due to the process not being ready when the sleep have finished. Maybe check the output of the server process to see when it is ready to receive queries. [Evaluate-ModelarDB-Changes/main.py](https://github.com/ModelarData/Utilities/blob/main/Evaluate-ModelarDB-Changes/main.py#L93) currently uses a simplified version of this approach as it sleeps while waiting for the nessesary output instead of making a blocking read on the stdout and/or stderr streams.

Reply:
Yes, I initially had what I thought would be a good fix [here](https://github.com/ModelarData/ModelarDB-RS/blob/f593aa5614b2932691252c343c377b9b7a4cd942/server/tests/integration_test.rs#L350). 

The problem was that I could not find a way to close `process::Stdio::piped()`, which resulted in the process becoming a zombie on Linux and not shutting down properly. This led to the next test not being able to start the server process. 

I think the 5 seconds of sleep should be enough for now, but if the problem with the checks arises again, I could revisit this. Although I think that should be a PR in and of itself.